modules/oaeservice/manifests/puppet.pp

```diff
@@ -7,4 +7,15 @@
   class { 'puppetdb::master::config':
     puppetdb_server => 'puppet',
   }
-}
\ No newline at end of file
+
+  # clean puppet report logs
+  # the report dir is not in puppet-hilary anywhere so extracted direct from puppet.conf
+  cron { 'clean-puppet-reports':
+    ensure  => present,
+    command => "find $(awk -F= '/^reportdir/ { print $2 }' /etc/puppet/puppet.conf) -type f -iname \*.yaml -ctime +30 -delete",
```

Comment:
I started there, but in production the reports are in /data/puppet/reports, hence the awk. I didn't check qa.
Honestly this is such a minor thing that I should just add the reportdir config into qa's puppet.conf.
I only noticed this issue 'cos puppet in staging is almost out of space, mostly these reports :)